homeassistant/components/tuya/select.py

```diff
@@ -75,6 +75,17 @@
             icon="mdi:thermometer-lines",
         ),
     ),
+    # Smart Water Timer
+    "sfkzq": (
+        # Irrigation will not be run within this set delay period
+        SelectEntityDescription(
```

Comment:
Move `name` into the translations as well.

Reply:
Now its using the translation_key